sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_HealthcareAsync_ShowStats.cs

```diff
@@ -0,0 +1,92 @@
+
+            List<string> batchInput = new List<string>()
+                {
+                    document,
+                    document,
+                };
+
+            AnalyzeHealthOptions options = new AnalyzeHealthOptions()
+            {
+                Top = 1,
+                Skip = 0,
+                ShowStats = true
+            };
+
+            AnalyzeHealthOperation healthOperation = await client.StartAnalyzeHealthAsync(batchInput, "en", options);
+
+            await healthOperation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection results = healthOperation.Value;
+
+            Console.WriteLine($"Results of Azure Text Analytics \"Healthcare Async\" Model, version: \"{results.ModelVersion}\"");
+            Console.WriteLine("");
+
+            foreach (RecognizeHealthcareEntititesResult result in results)
+            {
+                if (result.HasError)
+                {
+                    Console.WriteLine($"    Document error code: {result.Error.ErrorCode}.");
+                    Console.WriteLine($"    Message: {result.Error.Message}.");
+                }
+                else
+                {
+                    Console.WriteLine($"    Recognized the following {result.Entities.Entities.Count()} healthcare entities:");
+
+                    foreach (HealthcareEntity entity in result.Entities.Entities)
+                    {
+                        Console.WriteLine($"    Entity: {entity.Text}");
+                        Console.WriteLine($"    Subcategory: {entity.Subcategory}");
+                        Console.WriteLine($"    Offset: {entity.Offset}");
+                        Console.WriteLine($"    Length: {entity.Length}");
+                        Console.WriteLine($"    IsNegated: {entity.IsNegated}");
+                        if (entity.Links.Count > 0)
+                        {
+                            Console.WriteLine($"    Links:");
+                            foreach (HealthcareEntityLink healthcareEntityLink in entity.Links)
+                            {
+                                Console.WriteLine($"        ID: {healthcareEntityLink.Id}");
+                                Console.WriteLine($"        DataSource: {healthcareEntityLink.DataSource}");
+                            }
+                        }
+                    }
+
+                    Console.WriteLine($"    Document statistics:");
+                    Console.WriteLine($"        Character count (in Unicode graphemes): {result.Statistics.CharacterCount}");
+                    Console.WriteLine($"        Transaction count: {result.Statistics.TransactionCount}");
+                    Console.WriteLine("");
+                }
+            }
```

Comment:
For other endpoints, other type of request statistics: i.e. `documentCount`, `validDocumentsCount`.
It will be good to show that here too

Reply:
I am not sure if this is possible for this preview as adding it to TextAnalyticsStatistics class is causing it to be reflected in many other core files that I do not want to change. We Will update once we have something more stable.

Reply:
added